logging-modules/log4j/src/main/java/com/baeldung/reactiveLogging/WebFluxLogging.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.reactiveLogging;
+
+import reactor.core.publisher.Flux;
+
+public class WebFluxLogging {
+
+    public static void main(String[] args) {
+        Flux<Integer> reactiveStream = Flux.range(1, 5)
+            .log();
```

Comment:
I don't think that splitting the code across multiple lines makes sense in this file or `ReactiveLogging`. We sometimes follow this formatting in the article or when the fluent API calls will span more than one line, but they won't do that in either of these files

Reply:
Done.